src/main/java/com/ttbkk/api/domain/place/Place.java

```diff
@@ -0,0 +1,63 @@
+package com.ttbkk.api.domain.place;
+
+import com.ttbkk.api.BaseTimeEntity;
+import com.ttbkk.api.domain.brand.Brand;
+import com.ttbkk.api.domain.user.User;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.util.UUID;
+
+//Entity 클래스 . BaseTimeEntity 클래스를 상속 받아 자동 시간 생성.
+//User, Brand 클래스와 N:1 연관관계를 맺고 있어 해당 User, Brand 객체 필드로 가지고 있음.
+@Getter
+@NoArgsConstructor
+@Entity
+@Table(name = "place")
+public class Place extends BaseTimeEntity {
+
+    @Id
+    @Column(columnDefinition = "CHAR(32)")
+    private UUID id;
+
+    @NotNull
+    @Column(columnDefinition = "VARCHAR(100)")
+    private String name;
+
+//  위치의 경우 정확한 값을 요구하기 때문에 BigDecimal 타입 사용
+    @NotNull
+    @Column(columnDefinition = "DECIMAL(15,13)")
+    private BigDecimal latitude;
+    @NotNull
+    @Column(columnDefinition = "DECIMAL(15,12)")
+    private BigDecimal longitude;
```

Comment:
필드와 필드 사이에는 한줄 띄워주는게 좋을것같네요
```suggestion
//  위치의 경우 정확한 값을 요구하기 때문에 BigDecimal 타입 사용
    @NotNull
    @Column(columnDefinition = "DECIMAL(15,13)")
    private BigDecimal latitude;

    @NotNull
    @Column(columnDefinition = "DECIMAL(15,12)")
    private BigDecimal longitude;
```

Reply:
https://github.com/ttbkk-univ/ttbkk-spring/pull/7/commits/c7f0fafe667b6a65ea96724be268255f7465d7f3